PlaybackLab/ManUntdScrubberSampleApp/app/build.gradle

```diff
@@ -26,8 +27,8 @@ android {
 }
 
 task copyTask(type: Copy) {
-    from new File(vendorDir, 'Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
-    from new File(vendorDir, 'Ooyala/OoyalaSkinSDK-Android/OoyalaSkinSDK.jar')
+//    from new File(vendorDir, 'Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')

```

Comment:
why are this commented out? We won't be able to find the latest version of the SDKs